src/blockdb/sequential_files.cpp

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2015 The Bitcoin Core developers
+// Copyright (c) 2015-2018 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
this is a new file no need to include any copyright header earlier than 2018.

Reply:
I was unsure what to do about this. some of the functions in this file were moved from main.cpp which has the copyright for core. if I copy and pasted their code does credit need to be maintained or does it simply being a new file overwrite this need?

Reply:
IANAL I did not consider that code move was involved. Taking that into consideration I guess you did the right thing.